addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -41,6 +41,8 @@
 //- damagecustom          - Is damagecustom X, only works on 'attackdamage' and 'takedamage'
 //- backstabcount         - X consecutive backstabs required
 //- feigndeath            - Does client have feign death ready
+//- minimumdamage		  - Did client deal more than X amount of damage
+//- maximumdamage		  - Did client deal less than X amount of damage
```

Comment:
Think it would be better to have it named `damagemin`/`damagemax` so it would be consistent for `damagetype` and `damagecustom` having `damage` first in name, and `min`/`max` is named in params.
Same to enums etc in `tags_filter.sp`

Reply:
should be adjusted, hope i didn't miss anything in `tags_filter.sp`